examples/create-react-app-with-styled-components/README.md

```diff
@@ -22,8 +22,6 @@ npm start
 
 <!-- #default-branch-switch -->
 
-Note that CodeSandbox is not supporting react-app-rewired, yet you can [still see the code](https://codesandbox.io/s/github/mui-org/material-ui/tree/next/examples/create-react-app-with-styled-components).
-
 The following link leverages this demo: https://next.material-ui.com/guides/interoperability/#change-the-default-styled-engine with Parcel's alias feature within the `package.json`
 
 [![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/styled-components-interoperability-w9z9d)
```

Comment:
If codesandbox is now supported, we should update this link

x the other examples

Reply:
Oh ok, I have also tried a gist but it didn't work either https://codesandbox.io/s/beautiful-river-70kxs.